arcade/arcade/core/config_model.py

```diff
@@ -118,6 +118,14 @@ def load_from_file(cls) -> "Config":
 
         config_data = yaml.safe_load(config_file_path.read_text())
 
+        if config_data is None:
+            raise ValueError(
+                "Invalid credentials.yaml file. Please ensure it is a valid YAML file."
+            )
+
+        if "cloud" not in config_data:
+            raise ValueError("Invalid credentials.yaml file. Expected a 'cloud' key.")
```

Comment:
Unrelated -- added more graceful/informative error messages if the `credentials.yaml` file is borked for some reason